examples/zonal/bucket.tf

```diff
@@ -0,0 +1,44 @@
+# Creating the bucket, access controls and lifecycle
+resource "aws_s3_bucket" "bucket" {
```

Comment:
If the module created/managed the bucket we wouldn't have to do this in each example